lang/LangSource/Bison/lang11d

```diff
@@ -46,20 +46,20 @@ root	: classes
 
 classes : { $$ = 0; }
 		|	classes classdef
-			{ $$ = (long)linkNextNode((PyrParseNode*)$1, (PyrParseNode*)$2); }
+			{ $$ = (intptr_t)linkNextNode((PyrParseNode*)$1, (PyrParseNode*)$2); }
```

Comment:
Well, I cherry picked c0a550d to the 3.7 branch, it is only on master. That is also how the other stuff from master came in.